assets/js/element-shim.js

```diff
@@ -4,8 +4,6 @@
  * A temporary workaround to ensure the same version of React
  * is always used across multiple entrypoints.
  *
- * @private
```

Comment:
Why was this removed here?

Reply:
@felixarntz it was removed as there was a linter error generated from it. I couldn't find a work around, would you like me to look into getting the linter to ignore that line?